tests/material_tests.cc

```diff
@@ -440,4 +440,20 @@ TEST_F(MaterialTest, DecayHeatTest) {
   ASSERT_NEAR(3.614E-14 , dec_heat, 0.0005);
 }
 
+TEST_F(MaterialTest, GetNormalizedCompAtom) {
+  double val = 1.5 * units::kg;
+  Material::Ptr m1 = Material::CreateUntracked(val, diff_comp_);
+  cyclus::CompMap c = m1->comp()->atom();
+  cyclus::compmath::Normalize(&c);
```

Comment:
I think (?) the original pattern will result in the composition for material `m1` being changed because `atom()` returns a reference to that composition and `Normalize()` changes the values in place.  Maybe this is more `const` w.r.t. `m1`? 
```suggestion
  cyclus::CompMap c(m1->comp()->atom());
  cyclus::compmath::Normalize(&c);
```
It doesn't matter for testing purposes, but it may be useful to demonstrate a `const` pattern